.github/workflows/main.yml

```diff
@@ -57,6 +57,7 @@ jobs:
         run: pip install future mock pytest-coverage pytest-mock
 
       - name: Run Pytest and get code coverage for Codecov
+        if: ${{ matrix.python-version != '2.7' }}
```

Comment:
Only run python3 code coverage in feature/py3 branch.

Reply:
What I meant in my comment was to disable python2 tests for usb_scan.py, not the rest of the python code.

Also, I don't believe this is the way to remove python2 testing, this can be done at another level, also these makes the tests be run twice under python 3, check the step at line 71.